releng/com.espressif.idf.target/com.espressif.idf.target.target

```diff
@@ -16,17 +15,22 @@
 			<unit id="org.eclipse.help.feature.group" version="0.0.0"/>
 			<unit id="org.eclipse.platform.feature.group" version="0.0.0"/>
 			<unit id="org.eclipse.pde.feature.group" version="0.0.0"/>
-
+			<unit id="org.eclipse.sdk.feature.group" version="0.0.0"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<repository location="https://download.eclipse.org/releases/2025-03/"/>
+			<repository location="https://download.eclipse.org/releases/2025-09/"/>
 			<unit id="org.eclipse.cdt.autotools.feature.group" version="0.0.0"/>
 			<unit id="org.eclipse.cdt.feature.group" version="0.0.0"/>
 			<unit id="org.eclipse.cdt.sdk.feature.group" version="0.0.0"/>
 			<unit id="org.eclipse.cdt.cmake.feature.group" version="0.0.0"/>
 			<unit id="org.eclipse.tm.terminal.control.feature.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.tm.terminal.connector.process" version="0.0.0"/>
 			<unit id="org.eclipse.tm.terminal.feature.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.tm.terminal.view.ui" version="0.0.0"/>
 			<unit id="org.eclipse.tm.terminal.view.feature.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.ui" version="0.0.0"/>
+			<unit id="org.apache.commons.logging" version="0.0.0"/>
+			<unit id="javax.xml.jre" version="0.0.0"/>
```

Comment:
By default these will be included, do you see any specific usecase? if you need a specific version other than the default one for some reason, you can check eclipse orbit